packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -95,16 +91,14 @@ Obsoletes: %{?scl_prefix}rubygem-bastion
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.3.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.8.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 4.0.0
```

Comment:
This is an issue, since we have 4.x in RPMs. Will need an upstream PR as well.

Reply:
cc @sharvit